website/docs/ref/react.md

```diff
@@ -1,35 +1,31 @@
 # React API Reference
 
-Components from `@lingui/react` wrap the vanilla JS API from `lingui-i18n`. React components handle changes of active language or interpolated variables better than low-level API and also take care of re-rendering when wrapped inside pure components.
+Components from `@lingui/react` wrap the vanilla JS API from `@lingui/core`. React components handle changes of active language or interpolated variables better than low-level API and also take care of re-rendering when locale or messages change.
 
-## General Concepts
+## Rendering of Translations {#rendering-translations}
 
-### Rendering of Translations {#rendering-translations}
+All i18n components render translation as text without a wrapping tag. This can be customized in two different ways:
 
-All i18n components render translation as a text without a wrapping tag. This can be customized in three different ways:
-
--   globally: using `defaultComponent` prop on [`I18nProvider`](#i18nprovider) component;
--   locally: using `render` prop or `component` on i18 components
+-   globally: using `defaultComponent` prop on [`I18nProvider`](#i18nprovider) component
+-   locally: using `render` prop or `component` on i18n components
 
 #### Global Configuration
```

Comment:
Since the level of the heading was changed above, it would be great to change it here and below (consecutively increase the level)